Dockerfile

```diff
@@ -14,7 +9,7 @@ RUN make -j$(nproc) build
 
 # ACTUAL IMAGE -------------------------------------------------------
 
-FROM alpine:3.16
+FROM debian:12.1-slim
```

Comment:
I can only compare  alpine without RLN vs debian-slim with RLN, due to not being able to compile latest go-waku+RLN in alpine due to some incompaties due to musl (like [this](https://github.com/mattn/go-sqlite3/issues/1164)), so it's not a fair comparison due to not include the ~35mb from RLN.

But here's the results:
```
REPOSITORY           TAG       IMAGE ID       CREATED          SIZE      SHARED SIZE   UNIQUE SIZE   CONTAINERS
gowaku_alpine      latest    1a2424c5a4b8   25 seconds ago   62.6MB    0B            62.55MB       0
gowaku_debian      latest    e14ef7daacd6   5 minutes ago    191MB     0B            190.5MB       0
```

Alpine 62MB (I think it would be fair to assume 100MB in total with RLN)
vs 
Debian 191MB

I'd say some additional ~90mb, which seems okay